modules/netflow/configuration/logstash/netflow.conf.erb

```diff
@@ -206,6 +219,18 @@ filter {
         }
         
         # Populate packets transferred in the flow.
+        if [netflow][initiatorPackets] {
```

Comment:
Similarly here, these two clauses should be `else if`-joined and should be at the _tail_ of the `else if` chain.